docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2.3'
+version: '2.0'
```

Comment:
why downgrading the docker compose version?

Reply:
Running `docker-compse build` gives the following error:-
```
parth@parth:(master)~/snare$ docker-compose build 
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a version of "2" (or "2.0") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
```

hence I followed the suugestion

Reply:
@Parth1811 which version of Docker are you using?
Please check this
https://docs.docker.com/compose/compose-file/

I can't get an error you are reporting, so probably you have old docker version?

Reply:
@afeena  yes you are right, I was on an older version.